addons/website/views/website_templates.xml

```diff
@@ -593,7 +593,9 @@
 
 <!-- Language selector templates -->
 <template id="lang_flag" name="Language Flag">
-    <img class="o_lang_flag align-top" t-att-src="'/web/image/res.country/' + str(env.ref('base.' + lang_code.split('_')[1].lower()).id) + '/image'"/>
+    <t t-set="language" t-value="request.env['res.lang']._lang_get(lang_code)"/>
```

Comment:
@JKE-be if you can check the last commit of this PR too, especially this line: should we and how can we avoid it ?